roles/kafka_connect_replicator/tasks/health_check.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Get Authorization Token for Connect Replicator User
+  import_role:
+    name: common
+    tasks_from: get_authorization_token.yml
+  vars:
+    oauth: "{{ kafka_connect_replicator_oauth_enabled }}"
+    oauth_user: "{{ kafka_connect_replicator_oauth_user }}"
+    oauth_password: "{{ kafka_connect_replicator_oauth_password }}"
+    ldap_user: "{{ kafka_connect_replicator_ldap_user }}"
+    ldap_password: "{{ kafka_connect_replicator_ldap_password }}"
+  when: rbac_enabled or kafka_connect_replicator_oauth_enabled
```

Comment:
have we tested on non RBAC cluster with replicator oauth enabled?

Reply:
oauth + replicator rbac -> rbac-kafka-connect-replicator-kerberos-mtls-custom-rhel
no oauth + replicator rbac -> rbac-kafka-connect-replicator-kerberos-mtls-custom-ubuntu2004
no oauth + no rbac -> kafka-connect-replicator-mtls-scram-rhel

Havent tested oauth + no replicator rbac

Reply:
please update one of the replicator scenarios to test oauth on replicator on a non rbac cluster, I think kafka-connect-replicator-mtls-scram-rhel can be modified to test that